docs/profiling.md

```diff
@@ -64,37 +64,37 @@ Also, by default, the program will prompt you to open a link to
 file and open a visualizer. This feature is disabled by passing in
 `--no_perfetto_link` into the command. Alternatively, you can also point
 Tensorboard to the `log_dir` to analyze the trace (see the
-"Tensorboard Profiling" section below).
+"XProf (Tensorboard Profiling)" section below).
 
 (tensorboard-profiling)=
-## TensorBoard profiling
+## XProf (TensorBoard profiling)
 
-[TensorBoard's
-profiler](https://www.tensorflow.org/tensorboard/tensorboard_profiling_keras)
-can be used to profile JAX programs. Tensorboard is a great way to acquire and
+[XProf](https://www.tensorflow.org/tensorboard/tensorboard_profiling_keras)
+can be used to profile JAX programs. XProf is a great way to acquire and
 visualize performance traces and profiles of your program, including activity on
 GPU and TPU. The end result looks something like this:
 
-![TensorBoard profiler example](_static/tensorboard_profiler.png)
+![XProf example](_static/tensorboard_profiler.png)
 
 ### Installation
 
-The TensorBoard profiler is available as a plugin to TensorBoard
+XProf is available as a plugin to TensorBoard, as well as an independently
+ran program.
```

Comment:
Should we document how to run xprof independently?

Also nit: I think it should be "independently run" but not sure (or just "independent program")